handler_test.go

```diff
@@ -7,7 +7,9 @@ import (
 	"testing"
 	"time"
 
+	"fmt"
```

Comment:
sort these, all pkg ones at top and 3rd party separated by line space and at the bottom.